numpy/fft/helper.py

```diff
@@ -222,3 +224,46 @@ def rfftfreq(n, d=1.0):
     N = n//2 + 1
     results = arange(0, N, dtype=int)
     return results * val
+
+
+class _FFTCache(object):
+    """
+    Cache for the FFT init functions as an LRU (least recently used) cache.
```

Comment:
Should give this a standard docstring, see `doc/HOWTO_DOCUMENT.rst.txt`. All that is really needed here is a `Parameters` section with the two arguments to `__init__`.